test/functional/feature_evm_contracts.py

```diff
@@ -181,15 +217,168 @@ def failed_tx_should_increment_nonce(self):
 
         assert_equal(before_tx_count + 1, after_tx_count)
 
+    def should_deploy_contract_1KB_To_10KB(self):
+        node = self.nodes[0]
```

Comment:
Can set `self.node = self.nodes[0]` so that you don't have to repeat this in every test.

Reply:
Fixed: f4aff434c00d716b00880164e1428177d27be655